### PR TITLE
Fix authors name when only last name or no name given

### PR DESCRIPTION
**Pull request by ke4**
ebi-ait/dcp-ingest-central#364

Changes:
- When only last name given use that as name
- When no name given at all then use empty string as a name

### CODE REVIEW

File: src/app/projects/projects.service.ts [hunk 1]
```diff
@@ -62,16 +63,24 @@ export class ProjectsService {
...
+          if (author.hasOwnProperty('name')) {
+            const names = author.name.split(',');
+            formattedName = `${
+              names[names.length - 1]
+            } ${names[0][0].toUpperCase()}`;
+            fullName = author.name;
+          } else if (author.hasOwnProperty('last')) {
+            formattedName = author.last;
+            fullName = author.last;
+          }
```
Comment (aaclan-ebi) [inline src/app/projects/projects.service.ts:78]: this is conflicting with https://github.com/ebi-ait/projects-index/pull/57/files

Comment (ke4) [reply src/app/projects/projects.service.ts:78]: yep, that's why it is in Draft ;-)

Comment (ke4) [reply src/app/projects/projects.service.ts]: This now fixed

**Review by jacobwindsor — APPROVED** [review]
Thanks for adding tests!

**Review by MightyAx — APPROVED** [review]
LGTM

**Pull request merged**